src/content/plots/constants/Colours.tsx

```diff
@@ -6,15 +6,13 @@ export const colours = {
   axesColour: 'white',
   plotBorder: '1px solid black',
   colourPickerOptions: [
-    '#BC0000',
-    '#FF0074',
-    '#FF8921',
-    '#FFB61E',
-    '#069F00',
-    '#0088FF',
-    '#001BAA',
-    '#BF40BF',
-    '#919191',
-    '#000000'
+    'rgb(204, 121, 167)',
+    'rgb(213, 94, 0)',
+    'rgb(230, 159, 0)',
+    'rgb(240, 228, 66)',
+    'rgb(0, 158, 115)',
+    'rgb(86, 180, 233)',
+    'rgb(0, 114, 178)',
+    'rgb(0,0,0)'
```

Comment:
```suggestion
    '#0072B2',
    '#56B4E9',
    '#009E73',
    '#F0E442',
    '#E69F00',
    '#D55E00',
    '#CC79A7',
    '#000000'
```
The RGB values don't work correctly; they show correctly in the color picker but can't actually be applied to the plot. The values need to be hexadecimal.

The suggested change has also rearranged the colors as follows:

1. dark blue
2. light blue
3. green
4. yellow
5. light orange
6. dark orange
7. pink
8. black